### PR TITLE
Add Field To Display TVL (in USD) per Vault and Strategy

### LINKED ISSUE #34: Add total asset in usd value in vault page
Can use the yearn-sdk to get token want prices in USDC

https://github.com/yearn/yearn-sdk/blob/a411099f6d495abd7f967700c8d2340fe8f1dd0d/src/interfaces/earnings.ts#L335

Follow example

### PR DESCRIPTION
**Pull request by salazarguille**
It includes:

- React component to get the token price.
- Add the new component in the vault page and strategy page.

### CODE REVIEW

File: src/utils/oracle.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { getOracleInstance } from './abi';
+import BigNumber from 'bignumber.js';
+import { Token } from '../types';
+import { BigNumberish } from 'ethers';
+import { toUnits, USDC_DECIMALS } from './commonUtils';
+
+export const getTokenUnitPrice = async (token: Token): Promise<BigNumber> => {
```
Comment (storming0x) [inline src/utils/oracle.ts:7]: for later we will need to make this functions accept arrays

**Review by storming0x — APPROVED** [review]
LGTM!

File: src/utils/oracle.ts (hunk 1 above)
Comment (salazarguille) [reply src/utils/oracle.ts:7]: I will add it now.

Comment (salazarguille) [reply src/utils/oracle.ts:7]: Done!

**Pull request merged**